Gruntfile.js

```diff
@@ -252,32 +252,39 @@ module.exports = function(grunt) {
       configFiles: {
         files: [ 'Gruntfile.js', 'package.json' ],
         options: {
-          reload: true
+          reload: true,
+          interval: 1000
```

Comment:
I feel like we should be able to move the `interval` up to the parent level so we don't have to repeat it for every watch but I'm too lazy to try it out - can you give it a go?